src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -37,15 +37,15 @@
     },
     "core_eiaaeo__yearly_projected_generation_in_end_use_sectors_by_fuel_type": {
         "description": (
-            "Projected generation capacity & total generation in the end-use "
+            "Projected generation capacity and total generation in the end-use "
             "sector, broken out by fuel type.\n"
             "Includes combined-heat-and-power plants and electricity-only "
             "plants in the commercial and industrial sectors; and small on-site "
             "generating systems in the residential, commercial, and industrial "
             "sectors used primarily for own-use generation, but which may also "
             "sell some power to the grid. Solar photovoltaic capacity portion of "
-            "Renewable Sources in gigawatts direct current; other technologies "
-            "in gigawatts alternating current."
+            "Renewable Sources in megawatts direct current; other technologies "
+            "in megawatts alternating current."
```

Comment:
After the transformations these are in megawatts not gigawatts (to be consistent with all of our other data)